scripts/Steganography/decode.py

```diff
@@ -0,0 +1,6 @@
+from LSBSteg import *
+
+im = cv2.imread("my_new_image.png")
+steg = LSBSteg(im)
+print("Text value:", steg.decode_text())
+print("Text value:", steg.decode_text())

```

Comment:
Move this to `main.py` and make it so the user has the option to `encode` an image or `decode` an image.